config/attributes/clothing.json

```diff
@@ -1,33 +1,54 @@
 {
   "size": [
-    "0 (XXS)",
-    "2 (XS)",
-    "4 (S)",
-    "6 (S)",
-    "8 (M)",
-    "10 (M)",
-    "12 (L)",
-    "14 (L)",
-    "16 (XL)",
-    "18 (1X)",
-    "20 (2X)",
-    "22 (3X)"
+    "XXS",
```

Comment:
Instead of just listing all of them under "size", split into "shoe_size",
etc.  Not sure how many different types of sizing you have here, shoes,
clothes, anything else?

On Sat, Apr 25, 2015 at 7:45 PM, meaghanli <notifications@github.com> wrote:

> In config/attributes/clothing.json
> <https://github.com/howardc93/dukeexchange/pull/15#discussion_r29105301>:
>
> > @@ -1,33 +1,54 @@
> >  {
> >    "size": [
> > -    "0 (XXS)",
> > -    "2 (XS)",
> > -    "4 (S)",
> > -    "6 (S)",
> > -    "8 (M)",
> > -    "10 (M)",
> > -    "12 (L)",
> > -    "14 (L)",
> > -    "16 (XL)",
> > -    "18 (1X)",
> > -    "20 (2X)",
> > -    "22 (3X)"
> > +    "XXS",
>
> How do I do that?
> … <#14cf2f8d5cd79627_>
> On 25 April 2015 at 19:44, Howard Chung <notifications@github.com> wrote:
> In config/attributes/clothing.json <
> https://github.com/howardc93/dukeexchange/pull/15#discussion_r29105290>:
> > @@ -1,33 +1,54 @@ > { > "size": [ > - "0 (XXS)", > - "2 (XS)", > - "4
> (S)", > - "6 (S)", > - "8 (M)", > - "10 (M)", > - "12 (L)", > - "14 (L)", >
> - "16 (XL)", > - "18 (1X)", > - "20 (2X)", > - "22 (3X)" > + "XXS", You
> should probably separate each type of sizing (shoes, etc.) into its own key
> — Reply to this email directly or view it on GitHub <
> https://github.com/howardc93/dukeexchange/pull/15/files#r29105290>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/howardc93/dukeexchange/pull/15/files#r29105301>.
>